app.rb

```diff
@@ -0,0 +1,109 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author} "
+    end
+    nil
+  end
+
+  def all_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}"
+    end
+    nil
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)?: '
+    option = gets.chomp
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp
```

Comment:
Thanks for your feedback, we have created a method that is called request_option and now we are calling that method here instead of requesting the option directly in create_student. 📕